silk/datadog_checks/silk/events.py

```diff
@@ -0,0 +1,38 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+ALERT_TYPES = {
+    "INFO": "info",
+    "ERROR": "error",
+    # may have more?
+}
+
+
+class SilkEvent(object):
+
+    def __init__(self, raw_event, tags):
+        self.raw_event = raw_event
+
+        self.payload = {
+            "timestamp": self.raw_event['timestamp'],
+            "event_type": self.raw_event["labels"],
+            "source_type_name": 'silk',
+            "alert_type": ALERT_TYPES[self.raw_event["level"]],
+            "tags": tags[:],
+        }
+
+    def get_datadog_payload(self):
+
+        # Do any transformation here
+        self.transform()
```

Comment:
Not completely sure what transformations we may have yet

Reply:
Removing `transform()` for now since we need a call with the customer first to determine if it's necessary.